modules/motions/ui/MotionFormStartNew/Parts/StartSDVTNodeOperatorsAdd.tsx

```diff
@@ -82,17 +79,20 @@ export const formParts = ({
       nodeOperators: [
         { name: '', rewardAddress: '', managerAddress: '' },
       ] as NodeOperator[],
-      nodeOperatorsCount: [{ current: '' }] as { current: string }[],
+      nodeOperatorsCount: 'NaN',
```

Comment:
Is there a reason why should we declare `nodeOperatorsCount` as string of 'NaN'? I think we can declare it as a number (I'm sorry that I missed that moment in my previous comment) and use 0 (or `undefined as number | undefined`)

Reply:
updated,